pages/index.vue

```diff
@@ -60,7 +72,9 @@
             return;
           }
           return {
-            page: response.items[0].fields
+            page: response.items[0].fields,
+            isRichText: response.items[0].fields.hasPart[0].sys.contentType.sys.id === 'richText',
```

Comment:
These are always looking at the first hasPart but there may be multiple sections of different types.

`isRichText` and `isCardGroup` need to be methods taking one of the section objects as a parameter, then inspecting its `sys.contentType.sys.id`